internal/provider/models.go

```diff
@@ -150,3 +150,43 @@ type Script struct {
 type ScriptConfig struct {
 	Src string `json:"src"`
 }
+
+type ScopeCreate struct {
+	Scope Scope `json:"scope"`
+}
+
+type Scope struct {
+	Name        string                 `json:"name"`
+	Version     int                    `json:"version"`
```

Comment:
What's this version, I don't see it referenced anywhere else?

Reply:
Version is used on the `POST` body and without it the UI wasn't rendering as expected